CMakeLists.txt

```diff
@@ -4,6 +4,10 @@
 
 cmake_minimum_required(VERSION 3.5.1)
 
+if (NOT EXISTS ${CMAKE_SOURCE_DIR}/evmc/.git OR NOT EXISTS ${CMAKE_SOURCE_DIR}/cmake/cable/.git OR (TESTS AND NOT EXISTS ${CMAKE_SOURCE_DIR}/test/jsontests))
```

Comment:
I think moving this up this early will remove the `TESTS` variable.

I just propose we check here unconditionally for `evmc`, `cmake/cable` and `test/jsontests`.